lib/main/atom/commands/goToDeclaration.ts

```diff
@@ -56,3 +36,26 @@ commands.set("typescript:return-from-declaration", deps => {
       });
    }
 });
+
+export function handleDefinitionResult(result: protocol.DefinitionResponse,
+                                       location: FileLocationQuery): void {
+  if (result.body!.length > 1) {
```

Comment:
I agree. I just copied it, but then it was in a promise, so I guess we can argue the exception was handled :).

Fixed it now.